types/mysql__xdevapi/index.d.ts

```diff
@@ -0,0 +1,493 @@
+// Type definitions for @mysql/xdevapi 8.0
+// Project: https://github.com/mysql/mysql-connector-nodejs (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
It should be just `// Project: https://github.com/mysql/mysql-connector-nodejs` here